.gitignore

```diff
@@ -1 +1,4 @@
 vendor
+composer.lock
+.idea
```

Comment:
Please use a global gitignore locally. We don't care what IDE someone uses and you would want this ignored in all of your projects. (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)